core/templates/components/on-screen-keyboard/on-screen-keyboard.component.ts

```diff
@@ -29,8 +29,15 @@ import { GuppyInitializationService } from 'services/guppy-initialization.servic
   templateUrl: './on-screen-keyboard.component.html'
 })
 export class OnScreenKeyboardComponent {
-  engine;
-  guppyInstance;
+  engine!: {
```

Comment:
please add a comment explaining the use of non-null assertion

Reply:
done